pkgs/top-level/all-packages.nix

```diff
@@ -11292,7 +11292,10 @@ with pkgs;
     gtk3 = gtk4;
   };
 
-  webrtc-audio-processing_1 = callPackage ../development/libraries/webrtc-audio-processing { };
+  webrtc-audio-processing_2 = callPackage ../development/libraries/webrtc-audio-processing { };
```

Comment:
Add new package in the `all-packages` is not allowed now. Please move to `by-name`. See GitHub actions for details.